### PR TITLE
chore: remove deprecated field data from keyboard_info usage 🎺

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates to keymanapp/keyman#9351.

### CODE REVIEW

**Comment by mcdurdin** [comment]
Thought this was ready to go and then I looked at the schema again oops

File: tests/Search20Test.php [hunk 1]
```diff
@@ -113,23 +113,6 @@ public function testSearchByKeyboardId()
       $this->assertEquals('keyboard_id', $json->keyboards[0]->match->type);
     }
 
-    public function testSearchByLegacyKeyboardId()
```
Comment (darcywong00) [inline tests/Search20Test.php:116]: Does this get removed in a `Search30Test.php`?

Comment (mcdurdin) [reply tests/Search20Test.php:116]: Sorry, I don't understand the question?

Comment (darcywong00) [reply tests/Search20Test.php:116]: Search20Test.php uses /search/2.0/search.json
https://github.com/keymanapp/api.keyman.com/blob/be87e96ec10df73534eb758403e1fdcd88f8d55c/tests/Search20Test.php#L14C37-L14C61

Do we need a Search30Test.php that uses /search/3.0/search.json for tests?

Comment (mcdurdin) [reply tests/Search20Test.php:116]: No, because this is a test of the search 2.0 api, and we haven't got a search 3.0 api.

Comment (mcdurdin) [reply tests/Search20Test.php:116]: Oh wait

Comment (mcdurdin) [reply tests/Search20Test.php:116]: We've got a search 3.0 schema but not a search 3.0 api. Confuse much? Basically search 3.0 less data than search 2.0.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**